client/src/lib/utils.js

```diff
@@ -1,8 +1,10 @@
+import { flattenDeep, uniq } from 'lodash';
+
 /**
  *
  * @param {any[]} classes
  * @returns
  */
-export function createClassesFromArray(classes) {
-    return classes.filter(Boolean).join(' ');
+export function createClassesFromArray(...classes) {
+    return uniq(flattenDeep(classes)).filter(Boolean).join(' ');
 }
```

Comment:
What does this do exactly

Reply:
For adding multiple css classes easily in a readable and more maintainable way